### ISSUE TITLE
Problems with streaming audio

### ISSUE THREAD

**Issue by josea2019** [salvadordf/CEF4Delphi#126]
If you open your demo "SimpleBrowser2" and put the url "https://en-directo.frequence-radio.com/m80.html" you will notice that the audio is not heard, however, if you click on the "m80" link on that page, the audio plays correctly. In Chrome browser if you put the url "https://en-directo.frequence-radio.com/m80.html" you hear the audio perfectly without having to click on the link "m80".
Operating System: W10 Delphi XE2.
Tell me something about this problem please.
Is there a workaround to solve this problem?
Thank you very much for your help.
Best regards,
Jose A.Molina

**Comment by salvadordf** [comment]
It's probably a codec issue. The CEF binaries available at Spotify have the proprietary codecs disabled due to licensing issues and the US patent law. 

If you need to support those codecs for your app, all you need to do is build the CEF binaries following these instructions : http://magpcss.org/ceforum/viewtopic.php?f=6&t=13515

Check the radio stations in these sites and click on the MP3 streams (AAC is a proprietary codec and it won't work with the Spotify binaries) : 
http://www.radio-browser.info/gui/#/topclick
http://www.shoutcast.com/
https://tunein.com/radio/Spain-r100416/

**Comment by josea2019** [comment]
But it does not seem like a codec problem, if it were a codec problem, the audio would not be heard when the "m80" link is clicked. The problem only occurs when initially loading the page, not when clicking on the links of any of the radio stations.

**Comment by salvadordf** [comment]
Sorry, I answered too quickly. 

I checked that website with the MiniBrowser demo and the DevTools show several warnings and errors in the "Console" section. Chromium also shows those warnings but it keeps playing.

I also tried the [official CEF3 sample application](http://opensource.spotify.com/cefbuilds/cef_binary_3.3440.1805.gbe070f9_windows32_client.tar.bz2) with that website and it has the same problem. The player stops after 2-3 seconds.

However, this website plays that radio station correctly : http://www.emisora.org.es/m80/

There seems to be a problem with the media player included in that website and CEF seems to be less tolerant than Chromium. 

Perhaps it can be avoided using some switch from this website : 
https://peter.sh/experiments/chromium-command-line-switches/

Use GlobalCEFApp.AddCustomCommandLine with those switches before the GlobalCEFApp.StartMainProcess call.
